apps/opik-backend/src/main/resources/liquibase/db-app-state/migrations/000002_create_usage_usage_information_table.sql

```diff
@@ -0,0 +1,19 @@
+--liquibase formatted sql
+--changeset thiagohora:create_usage_usage_information_table
+
+CREATE TABLE metadata (
+    `key` VARCHAR(255) NOT NULL,
+    value VARCHAR(255) NOT NULL,
+    last_updated_at TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6),
+    PRIMARY KEY `metadata_pk` (`key`)
+);
+
+CREATE TABLE usage_information (
```

Comment:
The main purpose of this table is just to prevent the event from happening multiple times and to persist some information, such as the `anonymous_id` if we weren't using the `metadata` table. But we need to make sure that this doesn't really track the events, nor the actual usage information, as each local installation will have it's own.

I think it's missing some information, but that isn't that bad. The main problem is that events are only append only, but this table is mutable. It shouldn't be like that.

Also, the field names need to follow the usual convention of the service: `created_at` etc.